llvm/test/Transforms/LoopVectorize/scalable-first-order-recurrence.ll

```diff
@@ -197,7 +197,6 @@ define i32 @extract_second_last_iteration(ptr %cval, i32 %x)  {
 ; CHECK-VF4UF2: vector.ph
 ; CHECK-VF4UF2: call i32 @llvm.vscale.i32()
 ; CHECK-VF4UF2: call i32 @llvm.vscale.i32()
-; CHECK-VF4UF2: call i32 @llvm.vscale.i32()
 ; CHECK-VF4UF2: %[[VSCALE1:.*]] = call i32 @llvm.vscale.i32()
```

Comment:
Still some redundancies remain.

Reply:
Yes, there are a number of places remaining that re-compute runtime VF, will update those in follow-ups to use the VF VPValue.